### PR TITLE
Documentation: add additional information about Django admin remaining accessible with `PAPERLESS_DISABLE_REGULAR_LOGIN`

### PR DESCRIPTION
**Pull request by deviantintegral**
I understand from https://github.com/paperless-ngx/paperless-ngx/pull/5816 that this can’t or won’t be implemented in Paperless itself. That’s fine, but I think we should document this and offer an example of how someone could block it at a different layer in the stack.

I’ve set dev as the base branch as this feature hasn’t actually been released yet.

## Type of change

- [ ] Bug fix: non-breaking change which fixes an issue.
- [ ] New feature / Enhancement: non-breaking change which adds functionality. _Please read the important note above._
- [ ] Breaking change: fix or feature that would cause existing functionality to not work as expected.
- [x] Documentation only.
- [ ] Other. Please explain:

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [x] If applicable, I have included testing coverage for new code in this PR, for [backend](https://docs.paperless-ngx.com/development/#testing) and / or [front-end](https://docs.paperless-ngx.com/development/#testing-and-code-style) changes.
- [x] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [x] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [x] I have made corresponding changes to the documentation as needed.
- [x] I have checked my modifications for any breaking changes.

### CODE REVIEW

**Comment by deviantintegral** [comment]
Two notes: I’m on my iPad, so I made this change through the web UI and I haven’t run any pre commit hooks. If any CI jobs fail I’ll take a look.

Also, I wanted to say how much I appreciate the OIDC feature! It strikes me as much more robust than the forward-auth mode 👏 .

**Comment by deviantintegral** [comment]
What do you think about moving the note about blocking at a reverse proxy there? I think as written, a user may interpret that as "this route is required for Paperless to work" not "you'll have to use other tools to block this route".

**Comment by shamoon** [comment]
Sure that makes sense. I'll reopen this and you can change it as such, thanks.

One note is if you can remove the example nginx config, people use all kinds of RPs and in general we try to maintain generalizability. There's also a wiki with info about specific RPs.

**Review by shamoon — CHANGES_REQUESTED** [review]
https://github.com/paperless-ngx/paperless-ngx/pull/5993#issuecomment-1977104312 / https://github.com/paperless-ngx/paperless-ngx/pull/5993#issuecomment-1977140704

File: docs/advanced_usage.md [hunk 1]
```diff
@@ -695,4 +695,10 @@ More details about configuration option for various providers can be found in th
 
 ### Disabling Regular Login
 
-Once external auth is set up, 'regular' login can be disabled with the [PAPERLESS_DISABLE_REGULAR_LOGIN](configuration.md#PAPERLESS_DISABLE_REGULAR_LOGIN) setting.
+Once external auth is set up, 'regular' login can be disabled with the [PAPERLESS_DISABLE_REGULAR_LOGIN](configuration.md#PAPERLESS_DISABLE_REGULAR_LOGIN) setting. This setting will not black access to the built-in Django login form at `/admin/login`. To block access to that page, consider blocking it in your web server configuration. For example, with Nginx, try:
```
Comment (bogdal) [inline docs/advanced_usage.md:698]: Small typo
```suggestion
Once external auth is set up, 'regular' login can be disabled with the [PAPERLESS_DISABLE_REGULAR_LOGIN](configuration.md#PAPERLESS_DISABLE_REGULAR_LOGIN) setting. This setting will not block access to the built-in Django login form at `/admin/login`. To block access to that page, consider blocking it in your web server configuration. For example, with Nginx, try:
```

**Comment by shamoon** [comment]
@deviantintegral just clarifying that this is awaiting the changes we discussed before (see the “review”, not just the typo). If you’re not able to make them I can take care of it so we can close this out, let us know. Thanks

**Comment by deviantintegral** [comment]
Done! I also linked to the wiki docs.

One weird thing about that; the page had been completely replaced with just `rrr`. I reverted it, but what's odd is that the changes in the GitHub web UI don't match the underlying repo. I don't think there's anything to do, unless we notice other wiki pages have corrupt history.

https://github.com/paperless-ngx/paperless-ngx/wiki/Using-a-Reverse-Proxy-with-Paperless-ngx/_history

image

**Review by shamoon — APPROVED** [review]
Great thanks!

**Comment by shamoon** [comment]
Weird about the wiki thing. I’ll look into it thanks

**Pull request merged**